package.json

```diff
@@ -39,18 +40,25 @@
         "@babel/preset-flow": "^7.0.0",
         "@babel/preset-react": "^7.0.0",
         "@emotion/babel-preset-css-prop": "^10.0.7",
+        "babel-eslint": "^10.0.1",
         "babel-loader": "^8.0.5",
         "babel-plugin-emotion": "^10.0.7",
         "clean-css-cli": "4.2.1",
         "eslint": "5.1.0",
+        "eslint-config-prettier": "^4.0.0",
+        "eslint-plugin-flowtype": "^3.4.2",
+        "eslint-plugin-import": "^2.16.0",
         "eslint-plugin-jest": "22.1.2",
+        "eslint-plugin-jsx-a11y": "^6.2.1",
+        "eslint-plugin-prettier": "^3.0.1",
+        "eslint-plugin-react": "^7.12.4",
         "flow-bin": "^0.92.1",
         "gulp": "3.9.1",
         "gulp-sass": "4.0.1",
         "gulp-stylelint": "7.0.0",
         "gulp-watch": "5.0.0",
         "husky": "0.14.3",
-        "jest": "23.6.0",
+        "jest": "^24.1.0",
```

Comment:
@davidflanagan I tend to lean towards locking dependencies dow a specific version using `--save-exact` when installing. It seems we now have a mixture of locked down versions and "variable" versions. What are your thoughts on this?

I use Renovate on most of my other repos, not sure if we have the option of using it here. https://github.com/marketplace/renovate